modules/programs/nnn.nix

```
@@ -104,6 +104,38 @@ in {
         '';
         default = { };
       };
+
+      enableBashIntegration = mkOption {
+        default = true;
+        type = types.bool;
+        description = ''
+          Whether to enable Bash integration.
+        '';
+      };
```

All of these should be `mkEnableOption`

```suggestion
      enableBashIntegration = mkEnableOption "Bash integration" // {
        default = true;
      };
```